### PR TITLE
Coordinator live updates

### PR DESCRIPTION
**Pull request by yaron1m**

### CODE REVIEW

File: packages/common/src/types.ts [hunk 1]
```diff
@@ -117,3 +118,9 @@ export enum AppointmentStatus {
 }
 
 export const MANUAL_DONOR_ID = "manual";
+
+export type CoordinatorUpdate = {
```
Comment (ritabiniash) [inline packages/common/src/types.ts:122]: Maybe change to `CoordinatorUpdateProps` ? The current name implies that this is an action

File: packages/coordinator/src/store/appointments/actions/CoordinatorUpdateAction.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { CoordinatorUpdate } from "@zm-blood-components/common";
+import { ThunkAction } from "../../store";
```
Comment (ritabiniash) [inline packages/coordinator/src/store/appointments/actions/CoordinatorUpdateAction.ts:2]: What is `ThunkAction` ?

Comment (yaron1m) [reply packages/coordinator/src/store/appointments/actions/CoordinatorUpdateAction.ts:2]: Redux Thunk is a middleware for redux that lets you dispatch other actions and access the app state when running an action: https://github.com/reduxjs/redux-thunk

What I import here is the typing of thunk actions for our app (i.e. thunk + the structure of our state with appointments and stuff)

File: packages/common/src/types.ts (hunk 1 above)
Comment (yaron1m) [reply packages/common/src/types.ts:122]: I think `props` implies this is an object passed to a component.

How about `CoordinatorLastChangeDetails`?

File: packages/functions/src/coordinator/DeleteAppointmentsHandler.ts [hunk 3]
```diff
@@ -37,6 +38,8 @@ export default async function (
     await AppointmentDataAccessLayer.deleteAppointmentsByIds([appointmentId]);
   }
 
+  await setCoordinatorUpdate(appointment.hospital, callerId);
```
[the remark this answers is not in the document]
Comment (yaron1m) [reply packages/functions/src/coordinator/DeleteAppointmentsHandler.ts:41]: I thought about making this part of the props of `RequestHandler<T>` but I don't think it makes it much better.

Do you have any suggestions?

**Pull request merged**